qtpy/tests/conftest.py

```diff
@@ -21,8 +21,9 @@ def pytest_report_header(config):
     versions += 'PyQt6: '
 
     try:
-        from PyQt6 import Qt
-        versions += f"PyQt: {Qt.PYQT_VERSION_STR} - Qt: {Qt.QT_VERSION_STR}"
+        from PyQt6 import QtCore
+        versions += \
+            f"PyQt: {QtCore.PYQT_VERSION_STR} - Qt: {QtCore.QT_VERSION_STR}"
```

Comment:
I needed this on my system at least to get PyQt6 to actually show up

Reply:
Regarding this change, you didn't get shown the version info when testing locally? Could you try to restore this line and check if the issue is still presented (not seeing the version header in the pytest report) locally? Thanks!

Reply:
> Regarding this change, you didn't get shown the version info when testing locally? Could you try to restore this line and check if the issue is still presented (not seeing the version header in the pytest report) locally? Thanks!

If I revert this then I won't get the version:
```
$ python -c "from PyQt6 import Qt"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ImportError: cannot import name 'Qt' from 'PyQt6' (/home/larsoner/.local/lib/python3.10/site-packages/PyQt6/__init__.py)
```